json/src/main/scala/scalax/collection/io/json/imp/Stream.scala

```diff
@@ -186,11 +186,8 @@ object Stream {
               buf += {
                 val params: HyperEdgeParameters =
                   d.extract(jsonEdge).asInstanceOf[HyperEdgeParameters]
-                val nodeIds = params.nodeIds map lookupNode
-                var rest    = nodeIds
-                val _1      = rest.head; rest = rest.tail
-                val _2      = rest.head; rest = rest.tail
-                val _n      = rest
```

Comment:
The compiler actually just complained about the unused `_n`, but I went further used pattern matching to eliminate the `var` of `rest`.